Update searching-for-repositories.md

gusbo010 opened the pull request:
<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to a pull request) and what changes you've made, we can triage your pull request to the best possible team for review.

See our [CONTRIBUTING.md](/main/CONTRIBUTING.md) for information how to contribute.

For changes to content in [site policy](https://github.com/github/docs/tree/main/content/github/site-policy), see the [CONTRIBUTING guide in the site-policy repo](https://github.com/github/site-policy/blob/main/CONTRIBUTING.md).

We cannot accept changes to our translated content right now. See the [types-of-contributions.md](/main/contributing/types-of-contributions.md#earth_asia-translations) for more information.

Thanks again!
-->

### Why:

Closes #17388 
Discussion #17192
<!--
- If there's an existing issue for your change, please link to it.
- If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed:
content/search-github/searching-on-github/searching-for-repositories.md
Remove: **When you omit this qualifier, only the repository name and description are searched.**
<!-- Share artifacts of the changes, be they code snippets, GIFs or screenshots; whatever shares the most context. If you made changes to the `content` directory, a table will populate in a comment below with the staging and live article links -->

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

<!-- Description of the writer impact here -->

gusbo010 committed:
Update searching-for-repositories.md

ramyaparimi commented:
@gusbo010 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

kag728 commented:
Hi @gusbo010, thanks for opening this PR! It looks like you're right, when we don't specify an `in` qualifier, we default to search repository name, description and topics. Instead of removing the entire line, can we add in that topics are searched too? I have suggested the change on the commit. Thanks again!

gusbo010 committed:
Update content/search-github/searching-on-github/searching-for-repositories.md

Co-authored-by: Katherine Griffin <42548585+kag728@users.noreply.github.com>

kag728 commented on content/search-github/searching-on-github/searching-for-repositories.md:
```diff
@@ -23,7 +23,7 @@ To include forks in the search results, you will need to add `fork:true` or `for
 ## Search by repository name, description, or contents of the README file
 
-With the `in` qualifier you can restrict your search to the repository name, repository description, contents of the README file, or any combination of these. When you omit this qualifier, only the repository name and description are searched.
+With the `in` qualifier you can restrict your search to the repository name, repository description, contents of the README file, or any combination of these. When you omit this qualifier, only the repository name, description, and topics are searched.
```
```suggestion
With the `in` qualifier you can restrict your search to the repository name, repository description, contents of the README file, topics, or any combination of these. When you omit this qualifier, only the repository name, description, and topics are searched.
```

gusbo010 committed:
Update content/search-github/searching-on-github/searching-for-repositories.md

Co-authored-by: Katherine Griffin <42548585+kag728@users.noreply.github.com>

kag728 commented:
Hey @gusbo010! Thanks for waiting. This feature is fully rolled out. Before we merge this docs change, can we add `in:topics` as an example in the table below where the changes we've already made are? Once we add an example in, we'll be good to go here! Thanks 😄

skedwards88 reviewed (commented):
Thanks for this addition @gusbo010 ! Can you add an example to the table (lines 28-35) for `in:topic` (based on [this comment](https://github.com/github/docs/pull/17389#issuecomment-1156618517))? Once that is added, we'll merge this down for you.

gusbo010 committed:
Merge branch 'github:main' into main

gusbo010 committed:
Update searching-for-repositories.md

Example written under the assumption that in:topics uses exact match. Otherwise needs to be reworded.

gusbo010 commented:
@kag728, @skedwards88.
Example now added. The example I added is written using the assumption that in:topic matches exactly to specified query.
I.e. "java" matches topic "java" not topic "javascript". If this is not the case it should be reworded to something else.

skedwards88 reviewed (approved):
✨ Thank you! We'll get this merged down for you.

Commit:
Merge branch 'main' into main
